Aandachtspunten.md

```diff
@@ -1,9 +1,28 @@
 ## Aandachtspunten
 
+### Inhoud
+
 - Alle randvoorwaarden zijn voldaan.
 
-- Er wordt gebruik gemaakt van Dependency Injection via de constructor OF via @Autowired
+- Er wordt gebruik gemaakt van Dependency Injection via de constructor (OF via @Autowired, maar dat heeft niet de voorkeur
+
+- Er zijn Entities gemaakt voor Television, CiModule, WallBracket en RemoteController die inhoudelijk overeenkomen met de opdrachtbeschrijving
 
-- De service-laag verstuurd geen Television objecten naar de controller, enkel TelevisionDto’s
+- Er is een OneToOne relatie tussen Television en RemoteController. De relatie staat maar aan 1 kant, als de relatie aan 2 kanten staat (dus in beide klassen een @OneToOne), dan heeft een van beide kanten ook een "mappedBy"
```

Comment:
Bij de toevoeging van de regels met betrekking tot de relaties tussen de entiteiten (zoals OneToOne en ManyToMany), zorg ervoor dat je ook de bijbehorende methoden in de service laag implementeert. Dit is belangrijk voor de functionaliteit.